partner_tier_validation/data/tier_definition.xml

```diff
@@ -9,4 +9,15 @@
             name="definition_domain"
         >["&amp;",["parent_id","=",False],"|",["active","=",True],["active","=",False]]</field>
     </record>
+    <record id="partner_tier_definition_company_only" model="tier.definition">
+        <field name="name">Partner Validation (Company)</field>
+        <field name="model_id" ref="base.model_res_partner" />
+        <field name="review_type">group</field>
+        <field name="reviewer_group_id" ref="base.group_user" />
+        <field name="active">False</field>
```

Comment:
I feel it is OK to leave this, and have users explicitly enable it after installation.

Reply:
Since the other rule will be disabled, I will make this one the active one.